test/gzip_specials.js

```diff
@@ -7,16 +7,48 @@
 var fs      = require('fs');
 var path    = require('path');
 var assert  = require('assert');
+var zlib    = require('zlib');
 
 var pako_utils = require('../lib/utils/common');
 var pako    = require('../index');
 var cmp     = require('./helpers').cmpBuf;
+var constants = require('../lib/zlib/constants');
+
 
 
 function a2s(array) {
   return String.fromCharCode.apply(null, array);
 }
 
+function readMultiMemberStream(inputData) {
```

Comment:
I don't understand why to you use helper with chunks here instead of direct single `pako.inflate()` call.